app/apollo/test/cluster.fga.spec.js

```diff
@@ -0,0 +1,568 @@
+/**
+ * Copyright 2023 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+* Fine-Grained-Authorization (FGA) allows the auth model to determine access to resources based on the name or id of the resource, and on the action being taken (e.g. read vs edit).
+* This test suite provides validation of FGA behavior for auth models that support it by verifying that users with different FGA permissions are correctly allowed or restricted.
+* Other test suites will validate behavior based on having admin permissions (or lack thereof) independently.
+*/
+
+const { expect } = require('chai');
+const fs = require('fs');
+const Moment = require('moment');
+const { MongoMemoryServer } = require('mongodb-memory-server');
+const { v4: UUID } = require('uuid');
+
+const { models } = require('../models');
+const authApi = require('./api');
```

Comment:
Make variable naming consistent -- i.e. use `authFunc` here and then later
```
const authApi = authFunc(graphqlUrl);
const clusterApi = clusterFunc(graphqlUrl);
```
and then later again
```
    fgaToken01 = await signInUser(models, authApi, fgaUser01Data);
    fgaToken02 = await signInUser(models, authApi, fgaUser02Data);
```

Reply:
Done

Reply:
resolved